gpib_server.py

```diff
@@ -176,10 +189,26 @@ def read(self, c, n_bytes=None):
         Otherwise, reads until the device stops sending.
         """
         instr = self.getDevice(c)
-        if n_bytes is None:
-            ans = instr.read_raw()
+        if instr.resource_name.startswith('USB'):
+            try:
+                if n_bytes is None:
+                    ans = instr.read()
+                else:
+                    ans = instr.read(n_bytes)
+            except:
+                self.asynchronous_wait(0.1)
```

Comment:
@maffoo, okay it appears that everything works without the yield.  We tried with the yield and that ran into errors. The "read" method would likely need to use returnValue instead of return.  I am not sure what else.